src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -578,6 +592,18 @@ func (h *SkippedSequenceQueue) Remove(x uint64) error {
 
 }
 
+// Remove does a simple binary search to find and remove.
+func (h *SkippedSequenceQueue) Contains(x uint64) bool {
```

Comment:
The comment "Remove does a simple binary search to find and remove." seems misleading as the method name is "Contains"